playbooks/openshift-node/private/additional_config.yml

```diff
@@ -1,69 +0,0 @@
----
-- name: create additional node network plugin groups
-  hosts: "{{ openshift_node_scale_up_group | default('oo_nodes_to_config') }}:!oo_nodes_to_bootstrap"
-  tasks:
-  # Creating these node groups will prevent a ton of skipped tasks.
-  # Create group for flannel nodes
-  - group_by:
-      key: oo_nodes_use_{{ (openshift_use_flannel | default(False)) | ternary('flannel','nothing') }}
-    changed_when: False
-  # Create group for calico nodes
-  - group_by:
-      key: oo_nodes_use_{{ (openshift_use_calico | default(False)) | ternary('calico','nothing') }}
-    changed_when: False
-  # Create group for nuage nodes
-  - group_by:
-      key: oo_nodes_use_{{ (openshift_use_nuage | default(False)) | ternary('nuage','nothing') }}
-    changed_when: False
-  # Create group for contiv nodes
-  - group_by:
-      key: oo_nodes_use_{{ (openshift_use_contiv | default(False)) | ternary('contiv','nothing') }}
-    changed_when: False
-  # Create group for kuryr nodes
-  - group_by:
-      key: oo_nodes_use_{{ (openshift_use_kuryr | default(False)) | ternary('kuryr','nothing') }}
-    changed_when: False
-
-- import_playbook: etcd_client_config.yml
-  vars:
-    openshift_node_scale_up_group: "oo_nodes_use_flannel:oo_nodes_use_calico:oo_nodes_use_contiv:oo_nodes_use_kuryr"
-
-- name: Additional node config
```

Comment:
Just like the kuryr folks found out that their install method needs to change, we should probably let these other groups know these steps are no longer being performed.